From: Markus Lehtonen <markus.lehtonen@linux.intel.com>
Date: Tue, 23 Apr 2013 21:52:11 +0300
Subject: Makefile: configurable xsl stylesheet path for manpages

Makes it possible to build on different RPM based systems.

Signed-off-by: Markus Lehtonen <markus.lehtonen@linux.intel.com>
---
 Makefile.common  | 2 ++
 po4a/Makefile    | 3 +--
 scripts/Makefile | 4 +---
 3 files changed, 4 insertions(+), 5 deletions(-)

diff --git a/Makefile.common b/Makefile.common
index 3ffa4b4..ffafae2 100644
--- a/Makefile.common
+++ b/Makefile.common
@@ -6,3 +6,5 @@ GEN_MAN1S := bts.1 build-rdeps.1 chdist.1 dcontrol.1 debcheckout.1 debcommit.1 \
 PERLMOD_DIR = /usr/share/devscripts
 EXAMPLES_DIR = /usr/share/devscripts
 
+MAN_XSL_STYLESHEET ?= /usr/share/sgml/docbook/stylesheet/xsl/nwalsh/manpages/docbook.xsl
+
diff --git a/po4a/Makefile b/po4a/Makefile
index 6994bd8..df0add9 100644
--- a/po4a/Makefile
+++ b/po4a/Makefile
@@ -33,8 +33,7 @@ fr/%.fr.1: fr/%.fr.pl translate
 	podchecker $<
 	pod2man --utf8 --center=" " --release="Utilitaires Debian" $< > $@
 fr/%.fr.1: fr/%.fr.dbk translate
-	xsltproc --nonet -o $@ \
-	  /usr/share/sgml/docbook/stylesheet/xsl/nwalsh/manpages/docbook.xsl $<
+	xsltproc --nonet -o $@ $(MAN_XSL_STYLESHEET) $<
 # xsltproc writes out to foo.1 even if you tell it to do otherwise, so we have
 # to rename the file after it's generated
 	mv fr/$*.1 $@
diff --git a/scripts/Makefile b/scripts/Makefile
index a856bf5..ba04dcc 100644
--- a/scripts/Makefile
+++ b/scripts/Makefile
@@ -61,9 +61,7 @@ endif
 	podchecker $<
 	pod2man --utf8 --center=" " --release="Debian Utilities" $< > $@
 %.1: %.dbk
-	xsltproc --nonet -o $@ \
-	  /usr/share/sgml/docbook/stylesheet/xsl/nwalsh/manpages/docbook.xsl $<
-
+	xsltproc --nonet -o $@ $(MAN_XSL_STYLESHEET) $<
 # There is a slight chance this gets called twice, once here from here and once
 # from ../po4a/Makefile. Treat files with care.
 PID := $(shell echo $$$$-$$PPID)
